docs/how-to/aks.md

```diff
@@ -0,0 +1,247 @@
+# Running Krustlet in AKS
+
+These are steps for running a krustlet node in an AKS cluster. Ideally, we will
+want to simplify this process in the future so you do not have to do a bunch of
+configuration to connect a node to the cluster
+
+## Prerequisites
+A running AKS cluster (the steps below assume a non-custom vnet, though they are
+easily adaptable to custom vnets) and `kubectl` installed. See the [AKS
+docs](https://docs.microsoft.com/en-us/azure/aks/tutorial-kubernetes-deploy-cluster)
+for more information. Make sure you have the credentials to your cluster
+downloaded by running:
+
+```shell
+$ az aks get-credentials --name $CLUSTER_NAME --resource-group $RESOURCE_GROUP
+```
+
+This specific tutorial will be running krustlet on another Azure VM; however,
+you can follow these steps from any device that can access the internet.
```

Comment:
Not sure if this is quite accurate - the current setup requires a _device that can start a web server on an IP accessible from the Kubernetes control plane_ rather than _a device that can access the internet_, right?

Reply:
You are totally right. This has been fixed